### PR TITLE
New way to keep tax return cases used in the model

### PR DESCRIPTION
**Pull request by MalkIPP**

### CODE REVIEW

File: openfisca_france_data/input_data_builders/build_openfisca_survey_data/utils.py [hunk 1]
```diff
@@ -198,6 +199,24 @@ def check_structure(dataframe):
             "Wronger number of entity/head for {}".format(entity)
 
 
+def make_dic_var(year, cerfa_feuilles):
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_survey_data/utils.py]: Please find a better name ;-)

File: openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_05_foyer.py [hunk 2]
```diff
@@ -278,75 +278,11 @@ def foyer_all(temporary_store = None, year = None):
...
-        #Revenus à l'étranger
-        'f8ti': ['f8ti'],  # Revenus à l'étranger
-        'f8tl': ['f8tl'],  #
-        'f8tk': ['f8tk'],  #
-        'f8fv': ['f8fv'],  #
-        'f8tt': ['f8tt'],  #
-        'f8uu': ['f8uu'],  #
         }
-
+    cases_f6_f7_f8 = cerfa_field_by_foyer_entity(year = year, cerfa_feuilles = [6, 7, 8])
```
Comment (MalkIPP) [inline openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_05_foyer.py]: @benjello Is this better ?

**Comment by benjello** [comment]
Shouldn't you use make_cerfa_field_by_foyer_entity ?
Try to keep the same name in the function and in the code of the relevant step if possible.

File: openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_05_foyer.py [hunk 3]
```diff
@@ -278,75 +278,11 @@ def foyer_all(temporary_store = None, year = None):
...
-        #Revenus à l'étranger
-        'f8ti': ['f8ti'],  # Revenus à l'étranger
-        'f8tl': ['f8tl'],  #
-        'f8tk': ['f8tk'],  #
-        'f8fv': ['f8fv'],  #
-        'f8tt': ['f8tt'],  #
-        'f8uu': ['f8uu'],  #
         }
-
+    cases_f6_f7_f8 = make_cerfa_field_by_foyer_entity(year = year, cerfa_feuilles = [6, 7, 8])
```
Comment (benjello) [inline openfisca_france_data/input_data_builders/build_openfisca_survey_data/step_05_foyer.py]: Could you include a litte test of the inclusion of the previously explicit variables in your f6_f7_f8

**Comment by adrienpacifico** [comment]
close/merge ?

**Comment by benjello** [comment]
I will close it when in front of my computer ...

Le ven. 12 févr. 2016 20:04, adrienpacifico <notifications@github.com> a
écrit :

> close/merge ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openfisca/openfisca-france-data/pull/96#issuecomment-183452154>
> .
>
-- 
Mahdi Ben Jelloul

**Comment by benjello** [comment]
Integrated here: https://github.com/openfisca/openfisca-france-data/blob/master/openfisca_france_data/utils.py

**Pull request closed without merging**